standard/docs/en/getting_started/building_blocks.md

```diff
@@ -50,7 +50,7 @@ These are represented in a JSON document as follows:
 
 ### Building blocks: fields
 
-The OCDS schema sets out the fields that should be included in each section, making use of simple re-usable building blocks (field structures) to represent data. 
+The OCDS schema sets out the fields that can be included in each section, making use of simple re-usable building blocks (field structures) to represent data. 
```

Comment:
Perhaps this is one where it is better to keep the normative key word, to emphasise the fact that all fields in the schema are (at least) "should"

Reply:
The "should" has the caveat "where applicable" (to avoid overfill). We can add those to words here (and use "ought to", to avoid using a normative keyword in a non-normative section that repeats a normative section), or just leave it as "can". What do you prefer?

Reply:
I prefer "ought to" + "where applicable" and I opened an issue to consider how we can feature this more prominently in 1.1.5: https://github.com/open-contracting/standard/issues/856

Reply:
Fixed in c2b7578